backend/tests/unit/records/marshmallow/literature/test_bibtex_schema.py

```diff
@@ -492,3 +492,16 @@ def test_get_title_with_subtitle():
     result_title = result["title"]
 
     assert expected_title == result_title
+
+
+def test_bibtex_document_type_conference_paper_with_journal_title():
+    record = {"document_type": ["conference paper"],
+              "publication_info": [
+                  {"pubinfo_freetext": "Geneva, Switzerland: CERN (2002) 401 p"},
+                  {"pubinfo_freetext": "Geneva, Switzerland: CERN (2002) 401 p",
+                   "journal_title": "Nucl.Part.Phys.Proc."}
+              ]
+              }
```

Comment:
do you have [pre-commit hooks](https://pre-commit.com/) set up? black should run automatically and format the modified files. Formatting looks off here.